pyprocsync/__init__.py,sha256=9ybRXI3R8aJZGzhYdhI9tRVipHNMkuBCY_cTe6dUJRY,272
pyprocsync/exceptions.py,sha256=7tuK3g4LO10wQZpIc7qHZlWSJgLM-xvvKnhyBNqdZFc,632
pyprocsync/pyprocsync.py,sha256=RRexykkoy0zvJJiZGVC1iA2ZbZonpVrBbjPMMu9zmPw,7252
pyprocsync-0.1.0.dist-info/AUTHORS.rst,sha256=JdGiX86AYhf_LmKt97GRRUlqMjKSMDWyjc0N7lJl_nc,172
pyprocsync-0.1.0.dist-info/LICENSE,sha256=6sPZh4CAz2OV07Q3oeWnu8EIgK-aEDRKbr6kV22p1fg,1076
pyprocsync-0.1.0.dist-info/METADATA,sha256=fMb-ijKwlwLwAsdPiqL7gN_HXFNz-LIDR9_q2bfHppo,2561
pyprocsync-0.1.0.dist-info/WHEEL,sha256=Z-nyYpwrcSqxfdux5Mbn_DQ525iP7J2DG3JgGvOYyTQ,110
pyprocsync-0.1.0.dist-info/top_level.txt,sha256=t0fTkQw1wjUAbrPJLnmL5LGIOJTzj5FCAkNcGz8s7PM,11
pyprocsync-0.1.0.dist-info/RECORD,,
